src/Resources/contao/library/Contao/DcaExtractor.php

```diff
@@ -401,6 +401,12 @@ protected function createExtract()
 			return;
 		}
 
+		// Return if the DC type is "Folder" and is not database assisted
+		if ($GLOBALS['TL_DCA'][$this->strTable]['config']['dataContainer'] == 'Folder' && empty($GLOBALS['TL_DCA'][$this->strTable]['config']['databaseAssisted']))
```

Comment:
Probably just `$GLOBALS['TL_DCA'][$this->strTable]['config']['databaseAssisted']`? You have now added `isset()` to `databaseAssisted` but not to `dataContainer` …